### DISCUSSION TITLE
When I run the global model, there's an issue with the ID column in the returned results—it's not displaying correctly

### DISCUSSION THREAD

**Discussion by stanhaishan** [ourownstory/neural_prophet#1563] [Q&A - get help using NeuralProphet]
When I run the global model, there's an issue with the ID column in the returned results—it's not displaying correctly.
code:
m = NeuralProphet(
    trend_global_local="local",
    season_global_local="local",
    changepoints_range=0.8,
    trend_reg=5,
    n_forecasts=5,
    n_lags=20,
    unknown_data_normalization=True,
)
m.set_plotting_backend("plotly")

df_train, df_test = m.split_df(df1, valid_p=0.2, local_split=True) 
metrics = m.fit(df_train, freq='D')
future = m.make_future_dataframe(df2,n_historic_predictions=True) 
forecast = m.predict(future,decompose=False,raw=True)

![image](https://github.com/ourownstory/neural_prophet/assets/28746452/cc2ee2f0-6af5-4ce3-a81f-ebbf0639808d)

**Comment by iago-figueiredo-a3** [comment]
I got the same problem. It seems like it's a bug with the raw=True parameter. If you set raw=False, the ID column seems right. Then you have to shift back the yhat to get the step behavior of n-days-ahead prediction
